packages/component-library-react/src/Select.test.tsx

```diff
@@ -175,22 +158,14 @@ describe('Select', () => {
     });
 
     it('omits non-essential required attributes when not required', () => {
-      const { container } = render(<Select required={false} />);
```

Comment:
`required={false}` is needed here, we specifically want test that for attributes that are false by default we don't clutter the HTML.